contrib/experimental/great_expectations_experimental/expectations/expect_multicolumn_sum_values_to_be_between.py

```diff
@@ -0,0 +1,210 @@
+"""
+This is a template for creating custom MulticolumnMapExpectations.
+For detailed instructions on how to use it, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_multicolumn_map_expectations
+"""
+import functools
+import operator
+from typing import Optional
+
+from great_expectations.core.expectation_configuration import \
+    ExpectationConfiguration
+from great_expectations.exceptions import InvalidExpectationConfigurationError
+from great_expectations.execution_engine import (PandasExecutionEngine,
+                                                 SparkDFExecutionEngine,
+                                                 SqlAlchemyExecutionEngine)
+from great_expectations.expectations.expectation import \
+    MulticolumnMapExpectation
+from great_expectations.expectations.metrics.import_manager import F
+from great_expectations.expectations.metrics.map_metric_provider import (
+    MulticolumnMapMetricProvider, multicolumn_condition_partial)
+
+
+# This class defines a Metric to support your Expectation.
+# For most MulticolumnMapExpectations, the main business logic for calculation will live in this class.
+class ExpectMulticolumnSumValuesToBeBetween(MulticolumnMapMetricProvider):
```

Comment:
Let's just remove the `Expect` from the beginning of the metric name so that the metric and expectation have different class names (otherwise one will override the other). We typically use `the camel case version of the Metric Identifier, minus any periods.`

```suggestion
class MulticolumnValuesSumValuesToBeBetweenMaxAndMin(MulticolumnMapMetricProvider):
```